Ryujinx.HLE/HOS/Tamper/AtmosphereCompiler.cs

```diff
@@ -9,14 +9,36 @@ namespace Ryujinx.HLE.HOS.Tamper
 {
     class AtmosphereCompiler
     {
-        public ITamperProgram Compile(IEnumerable<string> rawInstructions, ulong exeAddress, ulong heapAddress, ITamperedProcess process)
+        private ulong    _exeAddress;
+        private ulong    _heapAddress;
+        private ulong    _aliasAddress;
+        private ulong    _aslrAddress;
+        ITamperedProcess _process;
```

Comment:
```suggestion
        private ITamperedProcess _process;
```
and realign var names

Reply:
Fixed